TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1982,6 +1978,17 @@ namespace
          }
       }
 
+      UTL_String get_filename()
```

Comment:
```suggestion
      UTL_String get_filename ()
```
Is this chunk broken into is own function so it can be called from somewhere else?  If so, it looks like managing the value of `n` should be handled somewhere else (constructor?) so that `get_filename ()` could be called multiple times on the same object and return the same result.